iroh-sync/src/actor.rs

```diff
@@ -522,6 +533,12 @@ impl SyncHandle {
         rx.await?
     }
 
+    pub async fn content_hashes(&self) -> Result<ContentHashesIterator> {
```

Comment:
Maybe add a comment that this is an iterator onto the actual redb table, which means writes are blocked until the iterator is dropped, right?

Reply:
I don't think this is true. According to the redb docs, "Read-only transactions may exist concurrently with writes".

It's a snapshot though, so you won't see writes of course.

![image](https://github.com/n0-computer/iroh/assets/248257/0096a137-8468-4f9c-a0b2-1e595c82d63f)